Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1218,6 +1225,79 @@ void MenuBar::GenerateSymbolsFromRSO()
   }
 }
 
+void MenuBar::GenerateSymbolsFromRSOAuto()
+{
+  constexpr std::array<std::string_view, 2> search_for = {".elf", ".plf"};
+  const AddressSpace::Accessors* accessors =
+      AddressSpace::GetAccessors(AddressSpace::Type::Effective);
+  std::vector<std::pair<u32, std::string>> matches;
+
+  // Find filepath to elf/plf commonly used by RSO modules
+  for (const auto& str : search_for)
+  {
+    u32 next = 0;
+    while (true)
+    {
+      auto found_addr = accessors->Search(next, reinterpret_cast<const u8*>(str.data()),
+                                          static_cast<u32>(str.size() + 1), true);
```

Comment:
This cast doesn't seem to be necessary any longer with the change over to size_t in the interface.